libs/experimental/langchain_experimental/llms/ollama_functions.py

```diff
@@ -30,10 +31,14 @@
 from langchain_core.runnables.passthrough import RunnablePassthrough
 from langchain_core.tools import BaseTool
 
-DEFAULT_SYSTEM_TEMPLATE = """You have access to the following tools:
+DEFAULT_SYSTEM_TEMPLATE = """You are a hyper intelligent AI assistant with access to the following tools:
```

Comment:
Changing default prompts like this can cause silent changes in behavior for users. Is this needed?

Would prefer to isolate this PR to the tool_calls attribute if possible.